Sources/MeiliSearch/Client.swift

```diff
@@ -164,6 +164,14 @@ public struct MeiliSearch {
     self.tasks.waitForTask(task: task, options: options, completion)
   }
 
+  public func waitForTask(
```

Comment:
I think you forgot to a comment above explaining the function and its parameters

Reply:
All the descriptions of the new classes/types/methods will be done after these PRs being merged :)

Reply:
They will not be in #321 ?